### DISCUSSION TITLE
Deployments stopped working

### DISCUSSION THREAD

**Discussion by nmsdvid** [vercel/vercel#6439] [Help]
Hello my deployments suddenly stopped working, my setup haven't changed it just stopped working. When trying to deploy I'm getting this error from Vercel, _Git author must have access to the project on Vercel_ Any ideas ?

**Comment by mcsdevv** [comment]
Hi @nmsdvid, this error occurs when the Git authors' email is not the same as the email address associated with the Vercel Personal Account.

I recommend checking the two to see which does not match, you can find information on doing so in [this support article](https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata).

**Reply by nmsdvid** [reply]
Hi @mcsdevv thanks for your reply, I have checked my credentials and they are the same. Any other ideas I could try?

**Reply by mcsdevv** [reply]
Hi there, this will certainly be the issue here. If you're having difficulty, contact us at vercel.com/support/request with an impacted SHA and we'll provide further assistance!

**Reply by nmsdvid** [reply]
thanks I have contacted support.
